grammars/php.cson

```diff
@@ -351,7 +351,7 @@
         'name': 'keyword.control.${1:/downcase}.php'
   }
   {
-    'begin': '(?i)\\b((?:require|include)(?:_once)?)\\s+'
+    'begin': '(?i)\\b((?:require|include)(?:_once)?)[\\s\\(]+'
```

Comment:
1. we want to check only for single parenthesis
2. don't consume parenthesis if you're not adding token for it
for the reasons above I'd suggest lookahead:
```suggestion
    'begin': '(?i)\\b((?:require|include)(?:_once)?)(?=\\s|\\()'
```
3. You need to add some testing to prevent regression